demo/depth.gdshader

```diff
@@ -0,0 +1,23 @@
+shader_type spatial;
+
+global uniform sampler2DArray environment_depth_texture;
+global uniform mat4 environment_depth_view_projection_left;
```

Comment:
Preferrably these would instead be array of matrices. But that does not seem to be possible, at least not in any way I have managed to find.